src/Uno.UI/Extensions/UIElementExtensions.cs

```diff
@@ -104,9 +104,6 @@ static string GetNumberedColumnIndentation(int depth)
 			elt switch
 			{
 				null => 0,
-#if NETSTANDARD
```

Comment:
I wonder if it should always be enabled. It's not super important though as it's a debug helper.

Reply:
Added back with corrected condition.